CHANGELOG.md

```diff
@@ -2,33 +2,46 @@
 
 All notable changes to this project will be documented in this file.
 
+## Release 0.4.0
```

Comment:
Learned from last time: can you also include the metadata.json update?

Reply:
Updated 👍